routes/leaguesearch.js

```diff
@@ -4,125 +4,144 @@ var http = require("http");
 var common = require('./common')
 var config = common.config();
 var _ = require('lodash'),
- 	mongojs = require("mongojs"),
-	db = mongojs.connect(config.mongo_uri);
+   mongojs = require("mongojs"),
+   async = require('async'),
+  db = mongojs.connect(config.mongo_uri);
 var quizCutoffDate = new Date()
 quizCutoffDate.setDate(quizCutoffDate.getDate() - 7);  //currently set to 1 week ago
 
 
 router.get('/', function(req, res) {
-	 db.collection('quiz').aggregate(
-
-
-	[{ "$match" : { "created_at" : { "$gt" : quizCutoffDate.toISOString().slice(0, 19).replace('T', ' ') }} //only pull quizzes in timeframe
-	},
-	{ "$group": {
-			"_id": {
-				"rb_team_id": "$rb_team_id",
-				"league": "$league",
-				"quiz_name": "$quiz_name"
-			},
-			"quizCount": { "$sum": 1}
-		}}
-
-	], function (err, result){ 
-		if (result.length > 0){
-			var all_leagues = createTeamLists(result);
-			var sorted_teams = sortTeams(all_leagues);  // Sort the teams within each league by number of quizzes taken
-		}
-
-		res.render('leaguesearch',
-			{ popular_teams: sorted_teams }
-		);		
-	}); 
-})
+
+  async.parallel({
+    goat_lists: fetchGoatLists,
+    leaders_lists: fetchLeadersLists
+  }, function(err,results){
+      console.log(results.goat_lists);
+      res.render('leaguesearch',
+        { popular_teams: results.goat_lists }
+    );  
```

Comment:
Are we not rendering the leaders list stuff just at this time or does it go somewhere else?